File: Code/App/v2/Laboratory/ViewModels/LabExaminationViewModel.cs
using BusinessLogic.Models;
using Laboratory.ViewModels;

namespace Laboratory.ViewModels
{
    public class LabExaminationViewModel
    {
        public PatientsModel PatientInfo { get; set; }
        public LabExaminationCommentViewModel LabComment { get; set; }
        public int PatientDieseaseId { get; set; }
    }
}

File: Code/App/separateDB/Laboratory/Scripts/App/Models/LabExaminationViewModel.js
function LabExaminationViewModel(data) {
    var self = this;

    this.PatientDieseaseId = ko.observable(data.PatientDieseaseId);

    this.LabComment = new
        LabExaminationCommentViewModel(data.LabComment);
    this.examinationDescriptionLab= ko.observable();

};